File: 00. Estructura/01. Lenguajes de proposito general/03. Persistencia/02. Archivos/Python/Archivos.py

# Abrir archivo
archivo = open(direccion, "r")     # En modo lectura (Solo prermite comandos de lectura)
archivo = open(direccion, "w")     # En modo escritura (Solo permite comandos de escritura) (Borra la información que habia en el archivo)
archivo = open(direccion, "a")     # En modo escritura (Solo permite comandos de escritura) (No borra la información que habia en el archivo)

# Cerrar archivo
archivo.close()

# Estructura para abrir, operar y cerrar el archivo
with open(direccion, "r") as file:    # En modo lectura
    # operaciones
with open(direccion, "w") as file:    # En modo escritura
    # operaciones

# Leer el siguientes caracter
cadena = archivo.read(1)

# Leer los siguientes n caracteres
cadena = archivo.read(n)

# Leer todo el archivo
cadena = archivo.read()

# Leer linea de caracteres
cadena = archivo.readline()

# Leer todas las lineas de caracteres
lista = archivo.readlines()

# Escribir cadena (Aañade la cadena a lo que ya esté escrito)
archivo.write(cadena)

# Nombre del archivo
archivo.name

# Modo del archivo ("r": lectura, "w": escritura)
archivo.mode

# Verificar si el archivo esta cerrado
archivo.closed

File: 01. Lenguajes de proposito general/03. Persistencia/02. Archivos de texto/C/Archivos.c

#include<stdio.h>

int main() {

    // Abrir archivo
    FILE *puntero = fopen("dirección", "r")     // En modo lectura (Solo prermite comandos de lectura)
    FILE *puntero = fopen("dirección", "w")     // En modo escritura (Solo permite comandos de escritura) (Borra la información que habia en el archivo) (Crea una archivo si no existe)
    FILE *puntero = fopen("dirección", "a")     // En modo escritura (Solo permite comandos de escritura) (No borra la información que habia en el archivo)
    FILE *puntero = fopen("dirección", "r+")    // En modo lectura y escritura (El archivo debe existir)
    FILE *puntero = fopen("dirección", "w+")    // En modo lectura y escritura (Borra la información que habia en el archivo) (Crea una archivo si no existe)
    FILE *puntero = fopen("dirección", "a+")    // En modo lectura y escritura (No borra la información que habia en el archivo)

    // Leer caracter del archivo
    fgetc(puntero)
    
    // Verificar si el caracter es fin de archivo
    fgetc(puntero) == EOF
    
    // Verificar si se puede escribir en un archivo
    !(puntero = fopen("dirección", "w"))
    
    // Escribir en el archivo
    fputs("texto", puntero)
    
    // Cerrar archivo
    fclose(puntero)

}